WordPressShared/WordPressShared/Core/Utility/NSString+Helpers.m

```diff
@@ -112,6 +112,14 @@ + (NSString *)stripShortcodesFromString:(NSString *)string
                                       withTemplate:@""];
 }
 
++ (NSString *)summaryFromContent:(NSString *)string
```

Comment:
@aerych I could get behind that. Let me move it into `createSummaryFromContent`.